packages/ng-schematics/src/schematics/ng-add/files/jasmine/e2e/support/jasmine.json

```diff
@@ -0,0 +1,9 @@
+{
+  "spec_dir": "e2e",
+  "spec_files": ["**/*[eE]2[eE].ts"],
+  "helpers": ["helpers/babel.js", "helpers/**/*.{js|ts}"],
+  "env": {
+    "stopSpecOnExpectationFailure": false,
+    "random": true
```

Comment:
do we need random? what does it do?

Reply:
also do we need to set stopSpecOnExpectationFailure?

Reply:
So random runs the test in a semi-random order, not really need for the single test. 
The second same with 1 test it does not matter.
Why I kept them - Jasmine's CLI creates a file similar to this one with these values pre applied, so decided to keep them.